README.md

```diff
@@ -1,43 +1,38 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Bookstore
 
-> One paragraph statement about the project.
+> Your "currently reading" book list, complete with your progress and running commentary, saved in an online API. Built with React-Redux.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./screenshot.png)
 
```

Comment:
- [ ] The link to the screenshot is broken. Kindly fix it so that the screenshot can be seen.